css/properties.json

```diff
@@ -5833,6 +5833,21 @@
     "order": "lengthOrPercentageBeforeKeywords",
     "status": "standard"
   },
+  "text-justify": {
+    "syntax": "auto | distribute | inter-character | inter-word | none",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Text"
+    ],
+    "initial": "auto",
+    "appliesto": "allElements",
```

Comment:
I've changed it to "inlineLevelAndTableCellElements", as this is the closest thing I could find that is used elsewhere in the file.